pkg/clustermesh/types/types.go

```diff
@@ -15,6 +15,18 @@ const (
 	ClusterIDMax = 255
 )
 
+func ValidateClusterID(clusterID uint32) error {
```

Comment:
is this change already part of https://github.com/cilium/cilium/pull/22875 ?

Reply:
Yes, but both PRs depend on this change, so let me rebase the branch after the one merged into master.